server/datastore/mysql/migrations/tables/20211202092042_RemovePolicyHistory.go

```diff
@@ -0,0 +1,49 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20211202092042, Down_20211202092042)
+}
+
+func Up_20211202092042(tx *sql.Tx) error {
+	_, err := tx.Exec("DROP VIEW policy_membership")
+	if err != nil {
+		return errors.Wrap(err, "drop table policy_membership_history")
```

Comment:
`drop view ...`

Reply:
Shouldn't we migrate the data instead? (or alternatively document on the release notes that policies may temporarily be missing after the upgrade)

Reply:
Yes! Great point.